gradle/wrapper/gradle-wrapper.properties

```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.3-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-2.6-bin.zip
```

Comment:
Final `2.7` will be out shortly, will push the `rc-2` until than (which has the fix for the [performance regression](https://docs.gradle.org/release-candidate/release-notes#important:-performance-regression-with-any-wrapper-generated-by-gradle-2.6))